packages/aws-cdk-lib/aws-codebuild/lib/project.ts

```diff
@@ -1965,6 +1965,8 @@ export class WindowsBuildImage implements IBuildImage {
   /**
    * The standard CodeBuild image `aws/codebuild/windows-base:2.0`, which is
    * based off Windows Server Core 2016.
+   *
+   * @deprecated `WindowsBuildImage.WIN_SERVER_CORE_2019_BASE_2_0` should be used instead.
```

Comment:
Added `@deprecated` to `WINDOWS_BASE_2_0`.
Since `WINDOWS_CONTAINER` type is already deprecated, a new deployment of `WINDOWS_BASE_2_0` will result in the following error.
`The environment type WINDOWS_CONTAINER is deprecated for new projects or existing project environment updates. Please consider using Windows Server 2019 instead.`

This is not the main topic of this PR, so if it should be a separate PR, I will do so.